tests/util/bip39_test_vectors_short.json

```diff
@@ -0,0 +1,124 @@
+{
+    "english": [
```

Comment:
Maybe just add the single new entry to the existing file?  or even generate it on load.